apertium-uzb.uzb.lexc

```diff
@@ -35708,7 +35740,7 @@ elfurush:elfurush N1 ; ! 0.9999999987189747
 elchilik:elchilik N1 ; ! 0.9999999995851189
 elchilik:elchilik N1 ; ! 0.9999999999999409
 elchixona:elchixona N1 ; ! "embassy"
-elchi:elchi N1 ; ! "postman"
+elchi:elchi N1 ; ! "ambassador"
 elchi:Elchi N1 ; ! "ambassador"
 Elchi:Elchi N1 ; ! "ambassador"
```

Comment:
Why do we need three versions of this?  Apertium should handle capitalisation differences automatically.

Reply:
Does it handle well?. Ex: names of the months in Uzbek were already there in capital letters, but I saw all month names in lowercases not-analysed when calculating uzb hfst coverage. So I was second-guessing about it.

Reply:
Noted. I fixed this and entries alike this one